lua/lspconfig/ui/lspinfo.lua

```diff
@@ -46,7 +53,18 @@ local function make_config_info(config)
   end
 
   local buffer_dir = vim.fn.expand '%:p:h'
-  config_info.root_dir = config.get_root_dir(buffer_dir) or 'NA'
+  local root_dir = config.get_root_dir(buffer_dir)
+  if root_dir then
+    config_info.root_dir = root_dir
+  else
+    config_info.root_dir = error_messages.root_dir_not_found
+    vim.list_extend(config_info.helptags, helptags[error_messages.root_dir_not_found])
+    local root_dir_pattern = vim.tbl_get(config, 'document_config', 'docs', 'default_config', 'root_dir')
```

Comment:
I guess this isn't entirely true should the user override the `root_dir` function, not sure how to handle that